libfwupd/meson.build

```diff
@@ -125,8 +128,7 @@ if get_option('introspection')
     export_packages : 'fwupd',
     header : 'fwupd.h',
     dependencies : [
-      giounix,
-      libcurl,
+      libfwupd_deps,
```

Comment:
this is actually adding more dependencies than needed

Reply:
Agree, good catch. Fixed.